### PR TITLE
Issue 13

### PR DESCRIPTION
**Pull request by gogobd**
I'm trying to fix testing now, too

### CODE REVIEW

File: plone-5.2.x.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://dist.plone.org/release/5.2.2-pending/versions.cfg
+    https://dist.plone.org/release/5.2.4/versions.cfg
```
Comment (ale-rt) [inline plone-5.2.x.cfg:3]: What about `https://dist.plone.org/release/5.2-latest/versions.cfg`?

File: requirements-5.2.x.txt [hunk 2]
```diff
@@ -1 +1 @@
--r https://dist.plone.org/release/5.2.2-pending/requirements.txt
+-r https://dist.plone.org/release/5.2.4/requirements.txt
```
Comment (ale-rt) [inline requirements-5.2.x.txt:1]: What about https://dist.plone.org/release/5.2-latest/requirements.txt?

**Review by ale-rt — APPROVED** [review]
LGTM, thanks!
Please check my comments if they make sense to you.

**Comment by gogobd** [comment]
@ale-rt Hey! Thanks for your suggestions :-)

**Review by ale-rt — APPROVED** [review]

**Pull request merged**